libbeat/cmd/keystore.go

```diff
@@ -0,0 +1,226 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	tml "golang.org/x/crypto/ssh/terminal"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/cmd/instance"
+	"github.com/elastic/beats/libbeat/common/terminal"
+	"github.com/elastic/beats/libbeat/keystore"
+)
+
+func getKeystore(name, version string) keystore.Keystore {
+	b, err := instance.NewBeat(name, "", version)
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error initializing beat: %s\n", err)
+		os.Exit(1)
+	}
+
+	if err = b.Init(); err != nil {
+		fmt.Fprintf(os.Stderr, "Error initializing beat: %s\n", err)
+		os.Exit(1)
+	}
+
+	return b.Keystore()
+}
+
+// GenKeystoreCmd initialize the Keystore command to manage the Keystore
```

Comment:
I have the `linter` on as I code, It used to be called `GenKeystoreCmd`, but I've renamed it since it doesn't need to be exported in any way. Not sure why my linter didn't catch it. 

I run:
```
  Enabled Linters: ['gofmt', 'golint', 'go vet']
```